frontend/src/components/pages/RegistrantExperience/RegistrationSteps/PersonalInfo/ContactCard.tsx

```diff
@@ -282,8 +282,6 @@ const ContactCard = ({
               <WrapItem
                 key={`contact_info_question_${question.question}`}
                 width={{ sm: "100%", md: mdWrapWidth }}
-                px="20px"
```

Comment:
Some minor styling fixes. Honestly, all the cards look quite wonky with all the additional questions we add (personalInfoQuestions in CamperCard, emergencyContactQuestions in ContactCard and their respective Edit variants). I've made another ticket to address making it look nicer, but that's quite low priority right now.

Reply:
sg